subsys/bluetooth/host/hci_core.c

```diff
@@ -21,12 +21,14 @@
 
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BLUETOOTH_DEBUG_HCI_CORE)
 #include <bluetooth/log.h>
+#include <common/log.h>
```

Comment:
There's nothing in our log.h API that should be public. The only reason it's in include/bluetooth is so that both drivers and subsys can access it. I suppose we could just move the existing one from include/bluetooth to subsys/bluetooth/common and add the necessary include lookup for drivers/bluetooth?